.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please remove this file and add to .ignore .vscode folder

Reply:
oks